db/migrate/20160401043112_create_phones.rb

```diff
@@ -5,7 +5,7 @@ def change
 
       t.string :number, null: false
       t.string :extension
-      t.string :type, null: false
+      t.string :service_type, null: false
```

Comment:
It was similiar to this

The single-table inheritance mechanism failed to locate the subclass: 'phones'. This error is raised because the column 'type' is reserved for storing the class in case of inheritance. Please rename this column if you didn't intend it to be used for storing the inheritance class or overwrite Project.inheritance_column to use another column for that information.